script/Ciel/Day 5/05_01_CIEL05_4B.txt

```diff
@@ -83,7 +83,7 @@ At this rate, my brain will shut down from this headache long before I breathe m
 [sha:435fb04c2f386e6ba567ea32204d46faf253a9c6]{
 -- Page 737, Offset 12852.
 -- 「そうですか。では、<ま|・><だ|・><信|・><じ|・><て|・>いいんですね？」
-"I see. Then, I can still count on you, can't I?"
+"I see. Then, I can still %{i}count%{/i} on you, can't I?"
```

Comment:
I couldn't think of any word to italicize here that doesn't just read weird in EN. Strictly it'd be "I can _still count_ on you can't I" but it's more confusing than useful so I'd just leave it dropped.